server/src/test/integration/org/apache/cassandra/sidecar/routes/tokenrange/LeavingTest.java

```diff
@@ -34,14 +34,7 @@
 
```

Comment:
`generateExpectedRangeMappingHalveClusterSize()` method is unused in this class and can be removed